agreement/gossip/network.go

```diff
@@ -87,6 +92,9 @@ func (i *networkImpl) processVoteMessage(raw network.IncomingMessage) network.Ou
 }
 
 func (i *networkImpl) processProposalMessage(raw network.IncomingMessage) network.OutgoingMessage {
+	if Trace != nil {
+		Trace.HashTrace("prop", raw.Data)
```

Comment:
could you add a string constant for "prop" in "github.com/algorand/go-algorand/network/messagetracer" ?
( just so we'll have a single location with all the constants we use for the hash trace )